src/pl/senla/hotel/comparators/RoomReservationsComparatorByGuestName.java

```diff
@@ -1,18 +1,19 @@
 package pl.senla.hotel.comparators;
 
+import pl.senla.hotel.application.annotation.GetInstance;
 import pl.senla.hotel.entity.Guest;
 import pl.senla.hotel.entity.services.RoomReservation;
 import pl.senla.hotel.repository.Repository;
-import pl.senla.hotel.repository.RepositoryGuestCollection;
 
 import java.util.Comparator;
 
 public class RoomReservationsComparatorByGuestName implements Comparator<RoomReservation> {
 
+    @GetInstance(beanName = "RepositoryGuestCollection")
```

Comment:
not the best approach to use the class name as bean name(how you will manage two beans of one class?), better to add a new field in AppComponent  annotation to specify a name if you need and if it's not specified then you can use class name by default, it's ok by now, but keep it in mind

Reply:
ok.
But we don't need manage "two beans of one class" because all our classes's beans are Singleton.